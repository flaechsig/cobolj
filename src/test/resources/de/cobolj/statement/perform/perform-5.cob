
IDENTIFICATION DIVISION.
PROGRAM-ID. HELLO.

DATA DIVISION.
   WORKING-STORAGE SECTION.
   01 WS-COUNTER PIC 9(1) VALUES 5.
   
PROCEDURE DIVISION.
    001-para.
        Display "001-para"
        Perform 003-para thru 005-para 2 times.
        
    002-para.
        display "002-para".
        Stop Run.
        
    003-para.
        display "003-para".
        
    004-para.
        display "004-para".
        
    005-para.
        display "005-para".