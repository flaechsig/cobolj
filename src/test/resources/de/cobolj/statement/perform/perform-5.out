001-para
003-para
004-para
005-para
003-para
004-para
005-para
002-para
